src/index.html

```diff
@@ -4,13 +4,57 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, 
+      user-scalable=no, 
+      initial-scale=1.0, 
+      maximum-scale=1.0, 
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <link 
+      rel="stylesheet"
+      type="text"
+      href="fonts/Avenir.ttc" 
+    >
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="form_big"
+      action="*" 
+      method="post" 
+      data-qa="big"
+    >
+      <img 
+        src="images/Search.svg" 
+        alt="big_icon" 
+        class="big_icon"
+      >
+      <input 
+        type="search" 
+        class="input" 
+        placeholder="Try &quot;Los Angeles&quot;" 
+        data-qa="keypress"
+      >
+    </form>
+
+    <form 
+      action="*" 
+      method="post"
+      class="form_small"
+      data-qa="small"
+    >
+      <img 
+        class="small_icon"
+        src="images/Search.svg" 
+        alt="Small" 
+      >
+      <input 
+        type="search" 
+        class="input--small" 
```

Comment:
```suggestion
      class="form"
      action="*" 
      method="post" 
      data-qa="big"
    >
      <img 
        src="images/Search.svg" 
        alt="big_icon" 
        class="icon"
      >
      <input 
        type="search" 
        class="input" 
        placeholder="Try &quot;Los Angeles&quot;" 
        data-qa="keypress"
      >
    </form>

    <form 
      action="*" 
      method="post"
      class="form form_small"
      data-qa="small"
    >
      <img 
        class="icon small_icon"
        src="images/Search.svg" 
        alt="Small" 
      >
      <input 
        type="search" 
        class="input input--small" 
```

Reply:
2nd point in the checklist

Reply:
If I names the class 'form form-small' or 'icon small-icon', the linter gives an error. Therefore, I styled as much  possible according to the checklist

Reply:
Small searchbar styles corrected. Duplicate styles removed